docs/test/endpoint-testing-guide.md

```
@@ -0,0 +1,191 @@
+
+# Specific test class
+mvn test -Dtest=AllergyServiceEndpointTest
+mvn test -Dtest=SystemInfoWsEndpointTest
+
+# Via make
+make install --run-unit-tests
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)